addon/modifiers/did-insert.js

```diff
@@ -46,7 +46,7 @@ import { setModifierManager, capabilities } from '@ember/modifier';
 */
 export default setModifierManager(
   () => ({
-    capabilities: capabilities('3.13', { disableAutoTracking: true }),
+    capabilities: capabilities('3.22', { disableAutoTracking: true }),
```

Comment:
Needs to use `gte` to use the capabilities as available by Ember version

See https://github.com/emberjs/ember-render-modifiers/pull/25#pullrequestreview-510614931